### PR TITLE
Bib

### PR DESCRIPTION
**Pull request by jvcasillas**
bib updates (about 90% done)

### CODE REVIEW

**Comment by jvcasillas** [comment]
We can (should) review and comment this before merging with main. @stefanocoretta @troettge

File: RR_manuscript/refs_to_add.md [hunk 1]
```diff
@@ -1,51 +1,9 @@
...
-
+Brugger 2001 (cant find)  
+Wagenmakers et al. 2012, (double check)  
+Wichert et al. 2015, (cant find)  
+Charles et al. 2019, (cant find)  
+Summerfield, 1984; (cant find)  
+Niebuhr et al., 2011 (cant find)  
+Haan 2002 (cant find)
+Gatt et al. 2013 (incomplete)  
+ogden2004 (is this 2009 or 2004?)
```
Comment (stefanocoretta) [inline RR_manuscript/refs_to_add.md:9]: 2004

**Comment by stefanocoretta** [comment]
Looks good to me!

**Comment by troettge** [comment]
Delete Summerfield from text. The rest is here:

@book{haan_speaking_2002,
	address = {Utrecht},
	title = {Speaking of questions},
	publisher = {LOT},
	author = {Haan, Judith},
	year = {2002}
}

@article{wagenmakers_agenda_2012,
	title = {An agenda for purely confirmatory research},
	volume = {7},
	number = {6},
	journal = {Perspectives on Psychological Science},
	author = {Wagenmakers, Eric-Jan and Wetzels, Ruud and Borsboom, Denny and van der Maas, Han LJ and Kievit, Rogier A.},
	year = {2012},
	pages = {632--638}
}

@incollection{brugger_haunted_2001,
	address = {Jefferson, NC},
	title = {From haunted brain to haunted science: {A} cognitive neuroscience view of paranormal and pseudoscientific thought},
	booktitle = {Hauntings and poltergeists: {Multidisciplinary} perspectives},
	publisher = {McFarland},
	author = {Brugger, Peter},
	editor = {Houran, James and Lange, Rense},
	year = {2001},
	pages = {195--213}
}

@article{wicherts_degrees_2016,
	title = {Degrees of freedom in planning, running, analyzing, and reporting psychological studies: {A} checklist to avoid p-hacking},
	volume = {7},
	journal = {Frontiers in psychology},
	author = {Wicherts, Jelte M. and Veldkamp, Coosje L. S. and Augusteijn, Hilde E. M. and Bakker, Marjan and Van Aert, Robbie and Van Assen, Marcel A. L. M.},
	year = {2016},
	pages = {1832}
}

@inproceedings{niebuhr_at_2011,
	address = {Hong Kong},
	title = {At the segment-prosody divide: {The} interplay of intonation, sibilant pitch and sibilant assimilation},
	booktitle = {Proceedings of the 17th {International} {Congress} of {Phonetic} {Sciences}},
	author = {Niebuhr, Oliver and Lill, Cassandra and Neuschulz, Jessica},
	year = {2011},
	pages = {1478--1481}
}

@article{charles_researcher_2019,
	title = {Researcher {Degrees} of {Freedom} in the {Psychology} of {Religion}},
	volume = {29},
	issn = {1050-8619},
	url = {https://doi.org/10.1080/10508619.2019.1660573},
	doi = {10.1080/10508619.2019.1660573},
	abstract = {There is a push in psychology toward more transparent practices, stemming partially as a response to the replication crisis. We argue that the psychology of religion should help lead the way toward these new, more transparent practices to ensure a robust and dynamic subfield. One of the major issues that proponents of Open Science practices hope to address is researcher degrees of freedom (RDF). We pre-registered and conducted a systematic review of the 2017 issues from three psychology of religion journals. We aimed to identify the extent to which the psychology of religion has embraced Open Science practices and the role of RDF within the subfield. We found that many of the methodologies that help to increase transparency, such as pre-registration, have yet to be adopted by those in the subfield. In light of these findings, we present recommendations for addressing the issue of transparency in the psychology of religion and outline how to move toward these new Open Science practices.},
	number = {4},
	urldate = {2021-01-20},
	journal = {The International Journal for the Psychology of Religion},
	author = {Charles, Sarah J. and Bartlett, James E. and Messick, Kyle J. and III, Thomas J. Coleman and Uzdavines, Alex},
	month = oct,
	year = {2019},
	note = {00009 
Publisher: Routledge
\_eprint: https://doi.org/10.1080/10508619.2019.1660573},
	pages = {230--245},
	file = {Charles et al_2019_Researcher Degrees of Freedom in the Psychology of Religion.pdf:/Users/timoroettger/Zotero/storage/S2R6YLCD/Charles et al_2019_Researcher Degrees of Freedom in the Psychology of Religion.pdf:application/pdf}
}

@article{gatt_models_2014,
	title = {Models and empirical data for the production of referring expressions},
	volume = {29},
	issn = {2327-3798},
	url = {https://doi.org/10.1080/23273798.2014.933242},
	doi = {10.1080/23273798.2014.933242},
	abstract = {This paper introduces a special issue of Language, Cognition and Neuroscience dedicated to Production of Referring Expressions: Models and Empirical Data, focusing on models of reference production that make empirically testable predictions, as well as on empirical work that can inform the design of such models. In addition to introducing the volume, this paper also gives an overview of recent experimental and modelling work, focusing on two principal aspects of reference production, namely, choice of anaphoric referential expression and choice of semantic content for referential noun phrases. It also addresses the distinction between dialogue and non-dialogue settings, focussing especially on the impact of a dialogue setting on referential choice and the evidence for audience design in the choices speakers make.},
	number = {8},
	urldate = {2021-01-20},
	journal = {Language, Cognition and Neuroscience},
	author = {Gatt, Albert and Krahmer, Emiel and Deemter, Kees van and Gompel, Roger P. G. van},
	month = sep,
	year = {2014},
	note = {00025 
Publisher: Routledge
\_eprint: https://doi.org/10.1080/23273798.2014.933242},
	keywords = {computational modelling, language production, referring expressions},
	pages = {899--911},
	file = {Gatt et al_2014_Models and empirical data for the production of referring expressions.pdf:/Users/timoroettger/Zotero/storage/4NMI33Z2/Gatt et al_2014_Models and empirical data for the production of referring expressions.pdf:application/pdf;Snapshot:/Users/timoroettger/Zotero/storage/VIYEK36R/23273798.2014.html:text/html}
}

**Pull request merged**